snooty/main.py

```diff
@@ -1,13 +1,12 @@
 """Snooty.
 
 Usage:
-  snooty build [--no-caching] <source-path> [<mongodb-url>] [--output=<path>] [options]
-  snooty watch [--no-caching] <source-path>
```

Comment:
Should this line be removed? For some reason, running the parser locally for me causes the following error:

```
File "/Users/raymundrodriguez/.pyenv/versions/3.10.4/lib/python3.10/site-packages/snooty/main.py", line 297, in main
    if args["watch"]:
KeyError: 'watch'
```

Re-adding this line seems to make the error go away. Are you experiencing this locally? Feel free to let me know if this is just an issue on my end!

Reply:
Oops! This line shouldn't have been removed 😅  Thanks!